WARNING: Your kernel does not support swap limit capabilities or the cgroup is not mounted. Memory limited without swap.
Traceback (most recent call last):
  File "/home/jdbl/core/Debloat.py", line 80, in depClean
    output = subprocess.check_output(cmd, shell=True)
  File "/usr/lib/python3.7/subprocess.py", line 395, in check_output
    **kwargs).stdout
  File "/usr/lib/python3.7/subprocess.py", line 487, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command 'cd /tmp/tmpvegrff2q/ice; mvn -gs /home/jdbl/core/../maven-proxy.xml se.kth.castor:depclean-maven-plugin:1.1.0:depclean -Dcreate.pom.debloated=true -Dcreate.result.json=true > /home/jdbl/results/JBEI_ice/be21f705e2503d730bd83d70df317d257c68c232/depClean.log 2>&1' returned non-zero exit status 1.
[ERROR] Command 'cd /tmp/tmpvegrff2q/ice; mvn -gs /home/jdbl/core/../maven-proxy.xml se.kth.castor:depclean-maven-plugin:1.1.0:depclean -Dcreate.pom.debloated=true -Dcreate.result.json=true > /home/jdbl/results/JBEI_ice/be21f705e2503d730bd83d70df317d257c68c232/depClean.log 2>&1' returned non-zero exit status 1.
OSError: [Errno 28] No space left on device

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "./bloat_report.py", line 45, in <module>
    json.dump(depclean, fd, indent=1)
OSError: [Errno 28] No space left on device
